swan-lake/learn/observing-ballerina-code.md

```diff
@@ -347,125 +346,9 @@ b7a.observability.tracing. jaeger.sampler.param | It is a floating value. Based
-    path = "/<absolute_path_to>/zipkin-2.6.1.jar"
-    groupId = "io.zipkin.reporter2"
-    modules = ["yourModuleName"]
-
-    [[platform.libraries]]
-    artifactId = "zipkin"
-    version = "2.5.0"
-    path = "/<absolute_path_to>/zipkin-reporter-2.5.0.jar"
-    groupId = "io.zipkin.zipkin2"
-    modules = ["yourModuleName"]
-
-    [[platform.libraries]]
-    artifactId = "zipkin-sender-okhttp3"
-    version = "2.5.0"
-    path = "/<absolute_path_to>/zipkin-sender-okhttp3-2.5.0.jar"
-    groupId = "io.zipkin.reporter2"
-    modules = ["yourModuleName"]
-
-    [[platform.libraries]]
-    artifactId = "zipkin-sender-urlconnection"
-    version = "2.5.0"
-    path = "/<absolute_path_to>/zipkin-sender-urlconnection-2.5.0.jar"
-    groupId = "io.zipkin.reporter2"
-    modules = ["yourModuleName"]
-
-    [[platform.libraries]]
-    artifactId = "kotlin-stdlib"
-    version = "1.3.31"
-    path = "/<absolute_path_to>/kotlin-stdlib-1.3.31.jar"
-    groupId = "org.jetbrains.kotlin"
-    modules = ["yourModuleName"]
-```
-
-7. Change the following configuration name to Zipkin. This ensures that all tracers are sent to Zipkin instead
-of the default Jaeger tracer.
-
-```toml
-[b7a.observability.tracing]
-name="zipkin"
-```
-
-8. The following configuration is a sample configuration option available for Zipkin tracer.
-
-```toml
-[b7a.observability.tracing.zipkin.reporter]
-hostname="localhost"
-port=9411
-
-[b7a.observability.tracing.zipkin.reporter.api]
-context="/api/v2/spans"
-version="v2"
-
-[b7a.observability.tracing.zipkin.reporter.compression]
-enabled=true
-```
-
-The table below provides the descriptions of each configuration option and possible values that can be assigned. 
-
-Configuration Key | Description | Default Value | Possible Values 
---- | --- | --- | --- 
-b7a.observability.tracing.zipkin. reporter.hostname | Hostname of the Zipkin server | localhost | IP or hostname of the Zipkin server. If it is running on the same node as Ballerina, it can be localhost. 
-b7a.observability.tracing.zipkin. reporter.port | Port of the Zipkin server | 9411 | The port that the Zipkin server is listening to.
-b7a.observability.tracing.zipkin. reporter.api.context | API context of the Zipkin server | `/api/v2/spans` | The API context of the Zipkin API. For V1 API, the context will be `/api/v1/spans`, and for V2 API, the context will be `/api/v2/spans` for default Zipkin server.
-b7a.observability.tracing.zipkin. reporter.api.version | API version of the Zipkin API | v2 | v1 or v2.
-b7a.observability.tracing.zipkin. reporter.compression.enabled | Enable the compression for the spans request | true | true or false.
-
 ### Setting Up the External Systems for Tracing
-Ballerina by default supports Jaerger and Zipkin for distributed tracing. This section focuses on configuring the
-Jaeger and Zipkin with Dockers as a quick installation.
+Ballerina by default supports Jaerger for distributed tracing. This section focuses on configuring the
```

Comment:
```suggestion
By default, Ballerina supports Jaeger for distributed tracing. This section focuses on configuring 
```

Reply:
Fixed in ad688a6a3d97f0d8b4988ad4d3f624ee746d91b2